GUIDELINES.md

```diff
@@ -37,11 +37,21 @@ No preset or field is isolated; they are always presented alongside others in va
 - ➕ **Suggested Additions**: Presets can suggest additional tags. These suggestions must be clearly supported by the wiki and community consensus.
 - 🔄 **Updates**: Deprecation rules can suggest updating tags. Good documentation and consensus are needed for these deprecations.
 
-In both cases, _indicators for consensus_ are:
+**In both cases, _indicators for consensus_ are:**
+
 - The deprecation is documented in the wiki and is either official (resulting from a proposal process) or long-standing (about a year).
 - There is a significant drop in usage compared to previous numbers, with a negative trend ([visible in the graph](https://taghistory.raifer.tech/)).
 - Usage of the deprecated tag remains stagnant for a longer period (about a year).
 
+**Deprecations are not for cleanup:**
+
+Deprecation rules work such that the user sees a message with suggestions and can act only when editing the given element. This makes them well-suited for gradual, human-reviewed updates of taggings like crossings. However, they are not suitable for cleaning up incorrect tagging from the database, especially for low-volume changes.
+
+There are, however, alternatives to consider: 
```

Comment:
```suggestion
There are, however, alternatives to consider:
- You can simply resurvey objects in your area, sometimes mapping based on aerial imagery and other resources available for OpenStreetMap mapping
- You can contact mappers (via changeset comments) and local communities (by creating notes, [writing on local community forums](https://wiki.openstreetmap.org/wiki/Contact_channels) and suggest looking at their local areas
```

Reply:
maybe also note that checking area distribution and sample of edits may be a good idea? sometimes it is a single edit or single mapper

sometimes it is even revertable as bogus import or vandalism

Reply:
I agree those are good things to do. But the cases when I link to this guideline to close issues or ask for clarification seem to not fit into this category of local or single user edits.

Maybe we should add a wiki page or a section on the organized editing guidlines to go into more details? This question of "how to cleanup values" should be common outside of this repo, right?

Reply:
> But the cases when I link to this guideline to close issues or ask for clarification seem to not fit into this category of local or single user edits.

ok, I was unaware of that, then it makes no sense

> Maybe we should add a wiki page or a section on the organized editing guidlines to go into more details? This question of "how to cleanup values" should be common outside of this repo, right?

+1 though I have no ready link - but if that case does not exist at problem at this repo we may simply not mention it